### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dreyhiflden**

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
+// Task 1
+/[0-2][0-9]:[0-5][0-9]/g
```
Comment (carrot-e) [inline script.js]: Check `29:00`

File: script.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+// Task 2
+/([-]?\d+|[-]?\d+\.\d+)[\s]?(\*|\/|-|\+)[\s]?([-]?\d+[\.]?\d+|[-]?\d+)/g
```
Comment (carrot-e) [inline script.js]: 1. Why do you use square brackets: `[-]`, `[\s]`, `[\.]`? What changes if you omit them?
1. `2   + 2` fails.

File: script.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+// Task 3
+function createURL(templateURL, params) {
```
Comment (carrot-e) [inline script.js]: The function should work correctly with arbitrary input:
```
createURL('/api/user/{userId}', {userId: 100});
```

Comment (dreyhiflden) [reply script.js]: Okey, fixed.

File: script.js (hunk 2 above)
Comment (carrot-e) [reply script.js]: 2.  Try several spaces: `2         + 2`

Comment (dreyhiflden) [reply script.js]: > 1. Try several spaces:
> 
> ```
> 2         + 2
> ```

Unexpected case. Fixed.

**Pull request merged**